doc/Uplink.xml

```diff
@@ -180,15 +192,64 @@
     <title>Uplink</title>
     <section>
       <title>Connection Establishment</title>
-      <para>The device initiates the connection to the cloud service. <xref linkend="_Ref505091745" /> shows the three phases. In the first phase the device acts as TLS client that connects to the cloud service. Please note that the figure only shows the most relevant packets. Details of the TCP and TLS exchange are out of scope of this specification. The second phase includes the connection upgrade to HTTP/2 which is confirmed by the cloud service with a 101 HTTP response. The third phase of the connection than fully complies to an HTTP2 connection as if it were initiated from the cloud service.</para>
-      <figure xml:id="_Ref505091745">
-        <title>Connection initiation from the device</title>
-        <mediaobject>
-          <imageobject>
-            <imagedata fileref="media/Uplink/image4.svg" contentwidth="128mm" />
-          </imageobject>
-        </mediaobject>
-      </figure>
+      <para>There are two connection mechanism available for a device to connect to a remote client. The device can either use an http/2 reverse tunnel or a WebSocket.</para>
```

Comment:
```suggestion
      <para>There are two connection mechanisms available for a device to connect to a remote client. The device can either use an http/2 reverse tunnel or a WebSocket.</para>
```

Reply:
@jflevesque-genetec @HansBusch updated my review with the use of the GitHub "Suggested" feature thanks JF :)